sources/packages/api/src/services/application/application.service.ts

```diff
@@ -402,4 +408,19 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .addOrderBy("application.applicationNumber")
       .getMany();
   }
+  /**
+   * get Student Application.
+   * @param applicationId application id.
+   * @param student student id.
+   * @returns student Application object.
+   */
+  async getStudentApplicationStatus(
```

Comment:
We already have a method that retrieves an application by user and id (getApplicationByIdAndUserName), is it possible to use it instead or maybe make it more generic and just rename it to getStudentApplication.